app-misc/ola/ola-0.10.7.ebuild

```diff
@@ -0,0 +1,100 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{6,7} )
+
+inherit autotools python-single-r1
+
+if [[ "${PV}" == "9999" ]]; then
+	inherit git-r3
+
+	EGIT_REPO_URI="https://github.com/OpenLightingProject/${PN}"
+fi
+
+DESCRIPTION="Open Lighting Architecture"
+HOMEPAGE="https://www.openlighting.org/"
+
+if [[ "${PV}" != "9999" ]]; then
+	SRC_URI="https://github.com/OpenLightingProject/${PN}/releases/download/${PV}/${P}.tar.gz"
+else
+	SRC_URI=""
+fi
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
```

Comment:
You can only add ~amd64 and ~x86 provided you tested your ebuild on them. Other keywords are handled by requesting keywording through bugzilla, where arch teams will add them.

Reply:
True. I'll remove ~arm and ~x86 and test the ebuild on my RPi 4 that runs ~arm64